### DISCUSSION TITLE
My Journey into Open Source

### DISCUSSION THREAD

**Discussion by felixfaisal** [EddieHubCommunity/support#1363] [Show and tell]
I started contributing around this time last year, I'd like to share you all how my journey has been so far :)) 
A good read for beginners starting out in open source 
[My Journey into Open Source](https://faisalahmedfarooq46.medium.com/my-journey-into-open-source-7ac74136cfd0)

**Comment by kaiwalyakoparkar** [comment]
I would love to suggest you some suggestion for writing the blog
- Divide 1 big paragraph into 2-3 small paragraphs.
- Use bold, underline, italic some specific keywords, etc.
- Add bulleted points to a paragraph
- Add emojis at the starting of every new heading or anywhere respectively.

ps - This are my view and suggestions

**Reply by kaiwalyakoparkar** [reply]
btw loved your blog

**Comment by eddiejaoude** [comment]
Awesome! 👍

**Comment by felixfaisal** [comment]
@kaiwalyakoparkar  Thanks for the feedback, I'll surely structure my content in the suggested way :rocket: 
@eddiejaoude Thanks for reading it :smiley:

**Reply by kaiwalyakoparkar** [reply]
Eager to see more awesome content from you! :heart:
